recipe/meta.yaml

```diff
@@ -50,7 +50,7 @@ requirements:
     - pluggy >=1.0.0
     - tqdm >=4
   run_constrained:
-    - conda-build >=3
+    - conda-build >=3.18.3
```

Comment:
@dholth Could you also send PRs for the feedstocks to make sure we're not missing this?